crates/common/consensus/lean/src/block.rs

```diff
@@ -5,21 +5,30 @@ use ssz_types::{VariableList, typenum::U4096};
 use tree_hash::TreeHash;
 use tree_hash_derive::TreeHash;
 
-use crate::attestation::SignedAttestation;
+use crate::attestation::Attestation;
 
-/// Represents a signed block in the Lean chain.
+/// Envelope carrying a block, an attestation from proposer, and aggregated signatures.
 ///
-/// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md#signedblock)
+/// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/5da200c13f5eeda0b4139b1d55970d75c011d4b2/src/lean_spec/subspecs/containers/block/block.py#L93)
```

Comment:
we should remove the link to the specification as this is inherently buggy and won't be maintained